binary_transparency/firmware/common/signature_test.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2020 Google LLC. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package common
+
+import (
+	"testing"
+
+	"github.com/golang/glog"
+)
+
+func TestSignatureVerification(t *testing.T) {
```

Comment:
Strictly, I think this is `TestSignatureRoundTrip` since you sign in here too :)

Reply:
ACK